brianlong/www.validators.app
# 186243184 stake account stats

#### What's this PR do?
- create stats for stake accounts in stake explorer and in vote accounts

#### How should this be manually tested?
- run migrations
- run tests
- populate explorer stake accounts with `rails r script/gather_explorer_stake_accounts.rb`
- check stake explorer page and vote accounts details - see if the stats are working as expected

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186243184)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
